conf/gear.conf

```diff
@@ -29,14 +32,20 @@ gearpump {
     masters = ["127.0.0.1:3000"]
   }
 
-
   ###########################
   ### Change the dispather for tasks
   ### If you don't know what this is about, don't change it
   ###########################
 
-  task-dispatcher = "akka.actor.pined-dispatcher"
-  netty-dispatcher = "akka.actor.default-dispatcher"
+  task-dispatcher {
+    mailbox-type = "akka.dispatch.SingleConsumerOnlyUnboundedMailbox"
+    throughput = 100
+    fork-join-executor {
+      parallelism-factor = 2
+      parallelism-max = 8
+      parallelism-min = 2
+    }
+  }
```

Comment:
The old behavior is to use pin-dispatcher, If there are multiple(suppose 1000 tasks), then the performance will be very bad(as it will try to create 1000 threads)

Now, the tasks will share the same thread pool.